lighthouse-core/config/lr-desktop-config.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/** @type {LH.Config.Json} */
+const config = {
+  extends: 'lighthouse:default',
+  settings: {
+    // disableStorageReset because it causes render server hang
+    disableStorageReset: true,
+    disableDeviceEmulation: true,
+    throttling: {
+      // Using a "broadband" connection type
```

Comment:
I ran a few tests with this vs. observed mode unthrottled. Observed tended to produce slightly more optimistic and variable results, so I'm inclined to say we keep this (then we should also be covered for that batch case @paulirish), but I don't feel strongly and would be fine with `throttlingMethod: 'observed'` as well.